index.js

```diff
@@ -0,0 +1,94 @@
+// Popup menu bar
+
+const body = document.querySelector('body');
+const navigation = document.querySelector('.nav-bar');
+const menu = document.querySelector('.logo');
+const menuSec = document.querySelector('.menu-bar');
+const closeButton = document.querySelector('#close-btn');
+const openButton = document.querySelector('#hamburg-open');
+const main = document.querySelector('main');
+
+const menuBar = () => {
+  menu.style.display = 'none';
+  navigation.style.boxShadow = 'none';
+  body.style.backgroundColor = '#F07F7E';
+  body.style.overflow = 'hidden';
+  menuSec.style.display = 'block';
+  menuSec.style.position = 'absolute';
+  menuSec.style.top = '160px';
+  menuSec.style.left = '50%';
+  menuSec.style.transform = 'translate(-50%, -50%)';
+  main.style.visibility = 'hidden';
+};
+openButton.addEventListener('click', menuBar);
+
+const closeMenu = () => {
+  body.style.overflow = '';
+  menu.style.display = '';
+  navigation.style.boxShadow = '';
+  body.style.backgroundColor = '';
+  menuSec.style.display = '';
+  main.style.visibility = '';
+};
+
+closeButton.addEventListener('click', closeMenu);
+
+// Section with featured speakers
+
+const presenters = [
+  {
+    featuredImage: 'images/steve-jobs.jpg',
+    name: 'Steve Jobs',
+    position: 'Founder of Apple inc',
+    shortBackground: 'With an idea of improving the means of communication in the world, he was ridiculed but eventually made the largest breakthrough in the mobile phone industry by creating the so called smart-phones',
+  },
+  {
+    featuredImage: 'images/bill.jpg',
+    name: 'Bill Gates',
+    position: 'Former CEO and founder of Microsoft Corporation',
+    shortBackground: 'Despite the option to return to school after major setbacks during the creation of Microsoft Corporation, he stilled pushed idea and eventually made it. His success made him the richest man for more than a century.',
+  },
+  {
```

Comment:
- [ ] You have done great implementing most of the requested changes from the previous reviewer. :+1: But there are still some changes you need to make.
- - As suggested by the previous reviewer, your Featured speaker section has 3 rows but the given design has just 2. I'll suggest you update your CSS to implement this as you are meant to replicate the given design. :smile: :wave: 

- - Your design:
 
![Selection_074](https://github.com/2004-okari/CAPSTONE-01/assets/107632780/90fe1cf3-108b-473d-8b44-1b7a077f6d64)

- - Given design:
![Screenshot from 2023-06-30 09-26-37](https://github.com/2004-okari/CAPSTONE-01/assets/107632780/5a74dd67-4a5a-4770-b9b5-9694881648e4)